DOCUMENTATION.md

```diff
@@ -60,8 +62,9 @@ All possible env vars are detailed in the [Config](https://godoc.org/github.com/
 type Config struct {
     // Verbosity is the logging verbosity: 0=panic, 1=fatal, 2=error, 3=warn, 4=info, 5=debug 6=trace
     Verbosity int `envvar:"VERBOSITY" default:"2"`
-    // DatabaseDir is the directory to use for persisting the database.
-    DatabaseDir string `envvar:"DATABASE_DIR" default:"./0x_mesh/db"`
+	// DataDir is the directory to use for persisting all data, including the
+	// database and private key files.
+	DataDir string `envvar:"DATA_DIR" default:"0x_mesh"`
```

Comment:
Is the alignment off here or is it just the diff that makes it look like that?

Reply:
It's off. Fixed!